utils/getScrollTop.js

```diff
@@ -4,5 +4,5 @@
  * @private
  */
 export default function getScrollTop () {
-  return Math.abs(parseInt(window.scrollY || window.scrollTop || document.getElementsByTagName('html')[0].scrollTop, 10))
+  return Math.abs((window.scrollY || window.scrollTop || document.getElementsByTagName('html')[0].scrollTop, 10))
```

Comment:
The braces and the radix from `parseInt()` need to be removed:
```suggestion
  return Math.abs(window.scrollY || window.scrollTop || document.getElementsByTagName('html')[0].scrollTop)
```